catalog/core/catalog-core-migratable/src/main/java/org/codice/ddf/catalog/migratable/impl/MigrationTaskManager.java

```diff
@@ -101,6 +135,7 @@ public void exportFinish() throws MigrationException {
             if (!isGracefulTermination) {
                 taskExecutor.shutdownNow();
             }
+            taskExecutor = MoreExecutors.listeningDecorator(executorSupplier.get());
```

Comment:
Looks like this is needed because we're trying to use the same instance for multiple exports, which isn't thread safe. May need to reconsider this (see my `ConfigurationMigratableImpl` comment above).